### ISSUE TITLE
VersionNumber needs non-Integers

### ISSUE THREAD

**Issue by SteveLLamb** [SMPTE/st429-16#9]
To match with the values used within the CTT, VersionNumber needs to be expanded to allow for non-integer values. As used by most content owners, features and trailers are often (more often than not) labeled as examples:
```
TLR-A, TLR-B, TLR-F1, TLR-F2
FTR-A, FTR-B, FTR-1, FTR-A1
```
With the current schema, we have to drop the letter used when both integers and letters are used, or default to a mapping of letter number to integer. These values should match as much as possible, as they are used for mastering and tracking purposes within many content and delivery databases.

**Comment by palemieux** [comment]
This will be a breaking schema change. Is the practice to use non-integer version number recent?

**Comment by SteveLLamb** [comment]
> This will be a breaking schema change. Is the practice to use non-integer version number recent?

Yes, agreed. And certainly not a new thing, I have CPLs going back to 2006 using letters in version.

**Comment by palemieux** [comment]
It would be good try to change the namespace and see if any common system fails.

**Comment by JBoisseau-ByDeluxe** [comment]
Another approach:
we add a new key "EditVersion"  = string
for trailers named as TLR-A1, EditVersion = "A", VersionNumber = 1

This "EditVersion" could also be used for Feature : shortened edit, long version, censor cuts, 
In that case, maybe we should categorize values, and avoid free form text.
